telegram/_bot.py

```diff
@@ -438,10 +438,15 @@ async def _send_message(
         data: JSONDict,
         reply_to_message_id: int = None,
         disable_notification: ODVInput[bool] = DEFAULT_NONE,
+        # To reduce repetition (if-clauses) in methods that call this method, following parameters
+        # are included separately instead of just passing them in data parameter:
+        # reply_to_message_id, reply_markup, message_thread_id, caption, caption_entities.
```

Comment:
that's the whole purpose of this method ;) Please either convert to a docstring or remove this comment.

Reply:
Sure, I get it. But in #3372 @harshil21 suggested that a comment be included in `Bot._send_message`. Since this method has no docstring at all, I decided that a comment was meant. If we're talking about a docstring, then maybe what was meant was that I should add something to a docstring of `Bot.send_message`, not `Bot._send_message`.

Reply:
Ah okay. Well, then let's do a docstring for `_send_message`, i.e. give the method an explanation about why it's there. Adding comments about `_send_message` to the docstring of `Bot.send_message` would not be appropriate since the latter is a public method and the user should not be concerned with the internals.

Reply:
I added a full-fledged docstring, not sure it was needed though because Sphinx doesn't render docstrings for protected methods (as far as I can tell).

Reply:
yep sphinx is not going to render this, this is documentation by devs for devs 😄

Reply:
Done